src/views/ItemView.js

```diff
@@ -86,11 +90,154 @@ import {
   WarningMessage,
   AdministrativeNoteList,
   ItemViewSubheader,
+  PaneLoading,
 } from '../components';
 
+const getRequestsPath = `circulation/requests?query=(itemId==:{itemid}) and status==(${requestsStatusString}) sortby requestDate desc&limit=1`;
```

Comment:
I understand that this is just code replaced from another component, but I don't see any reason to have a separate variable for this path, it's used in one place only and the name of the variable looks like this is a function rather than a constant. I suggest either putting it to manifest or renaming it.